xliff/lib.en-US.xlf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2">
+  <file original="lib.json" source-language="en-US" target-language="en-US" datatype="plaintext">
+    <body>
+      <trans-unit id="loc.messages.LIB_UnhandledEx">
```

Comment:
make a different prefix so it doesnt collide... task lib already register unhandled ex so this may not be required